modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -542,6 +520,8 @@ function Filters({fdsFilterClientExtensions, fdsView, namespace}: IProps) {
 				OBJECT_RELATIONSHIP.FDS_VIEW_FDS_DYNAMIC_FILTER
 			] as ISelectionFilter[];
 
+			// TODO: set a default filter name
+
```

Comment:
Does this mean we want to do the same here? (the same = use field name as the filter label and save it if no label is provided

Reply:
:grimacing: It is related to the other change and forgot to remove the comment